File: C/Iniciante/1013.c
#include <stdio.h>
#include <stdlib.h>

int main (void)
{
	int A, B, C, M1, M2;
	scanf ("%i %i %i", &A ,&B, &C);
	M1 = (A+B+abs(A-B))/2;
	M2 = (M1+C+abs(M1-C))/2;
	printf ("%i eh o maior\n", M2);

	return 0;
}

File: Python/Iniciante/1013.py
linha = input().split(" ")
a1, a2, a3 = linha
a = int(a1)
b = int(a2)
c = int(a3)

maiorAB = (a + b + abs(a - b)) / 2

maiorABC = (maiorAB + c + abs(maiorAB - c)) / 2

print("%d eh o maior" %(maiorABC))